source/documentation/using_ci/jenkins.md

```diff
@@ -7,6 +7,10 @@ There are two main approaches to pushing applications to GOV.UK PaaS with [Jenki
 
 Both approaches require you to set up the credentials plugin first.
 
+### Set up dedicated users for testing
+
+In order to avoid developer credentials - particularly those with admin rights - leaking, you should should request a dedicated PaaS user account for use by Jenkins. If you want to deploy to multiple spaces with Jenkins, use a different user account for each. Each user should only have `SpaceDeveloper` access to a single space within your organisation.
```

Comment:
Should this link to `#credentials-for-automated-accounts` (like the Travis section does) rather than duplicating the content?